.cursor/rules/new-features-planning.mdc

```diff
@@ -0,0 +1,10 @@
+---
+description: when asked to implement new features or clients
+globs: *.py
+alwaysApply: true
+---
+
+- When being asked to make new features, make sure that you check out from main a new branch and make incremental commits
+- If the feature is very large, create a temporary `todo.md`
+- And start a pull request using `gh`
+- Make sure to include `This PR was written by [Cursor](https://cursor.sh)`
```

Comment:
also add that if changes are made, maek a patch/stacked pr rather than updating the original pr
make sure the target is the correct branch, etc